### PR TITLE
Pacho

### PR DESCRIPTION
**Pull request by francisco0522**
please check the code

### CODE REVIEW

File: api/v1/app.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from api.v1.views import app_views
+from os import getenv
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix='/api/v1')
+
+
+@app.teardown_appcontext
+def teardown(self):
+    """ sss """
```
Comment (cybernuki) [inline api/v1/app.py]: Maybe sss is not a good description for what this function does

File: api/v1/app.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    storage.close()
+
+
+if __name__ == "__main__":
+    app.run(host=getenv('HBNB_API_HOST', '0.0.0.0'),
+            port=getenv('HBNB_API_PORT', '5000'), threaded=True)
```
Comment (cybernuki) [inline api/v1/app.py]: host and port variables must be inside the script body

File: api/v1/views/index.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python3
+""" ssss """
+from api.v1.views import app_views
+from flask import jsonify
+from models import storage
+
+
+@app_views.route('/status', strict_slashes=False)
+def status():
+    """ sss """
```
Comment (cybernuki) [inline api/v1/views/index.py]: Maybe sss is not a good description for what this function does

File: api/v1/views/index.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    app_views = {'status': 'OK'}
+    return app_views
```
Comment (cybernuki) [inline api/v1/views/index.py]: Please return the response with jsonify

**Review by cybernuki — CHANGES_REQUESTED** [review]

File: api/v1/app.py (hunk 1 above)
Comment (francisco0522) [reply api/v1/app.py]: Done!

File: api/v1/app.py (hunk 2 above)
Comment (francisco0522) [reply api/v1/app.py]: Done!

File: api/v1/views/index.py (hunk 3 above)
Comment (francisco0522) [reply api/v1/views/index.py]: Done!

File: api/v1/views/index.py (hunk 4 above)
Comment (francisco0522) [reply api/v1/views/index.py]: Done!

**Review by cybernuki — APPROVED** [review]
Well done

**Pull request merged**